data/json/mapgen/basecamps/expansion/modular_storehouse/version_1/modular_storehouse_common.json

```
@@ -13,13 +13,182 @@
         "      ",
         "      "
       ],
-      "palettes": [ "fbms_wood_palette" ]
+      "flags": [ "ALLOW_TERRAIN_UNDER_OTHER_DATA" ],
+      "palettes": [ "fbms_common_palette" ]
```

```suggestion
      "palettes": [ "fbms_common_palette" ],
```
Making change that I will revert to reset the tests.